.travis.yml

```diff
@@ -1,76 +1,68 @@
-language: python
+language: c
```

Comment:
because we don't need to use Travis's python envs, and they don't work on OS/X anyway.